lms/static/js/staff_debug_actions.js

```diff
@@ -48,6 +52,7 @@ var StaffDebug = (function() {
                 var html = _.template('<p id="idash_msg" class="success">{text}</p>', {interpolate: /\{(.+?)\}/g})(
                 {text: text}
             );
+                // xss-lint: disable=javascript-jquery-html
```

Comment:
Please fix these XSS violations. Normally, when we are doing fixes and come across the files that have XSS vulnerability in them, we tend to fix them instead of disabling them. You can visit the [documentation link](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/preventing_xss/preventing_xss.html#linter-violations) that has all the right steps for fixing them.

Reply:
As per documentation `HtmlUtils.setHtml()` is recomended to use instrad of `html()` but `HtmlUtils.setHtml()` sets the HTML as `string` which cannot be rendered on screen. which is not required behaviour . So have to supress this warning .

Reply:
This exception is happening due to the element, being passed in $.html(), not containing any HTML tag. The fix suggests to convert it to string or use `setHtml` method. If the data were to contain HTML, the warning wouldn't have been raised.

Reply:
We need to validate if the `html` variable is actually HTML. If it is, the warning shouldn't come. Even without that, the edxUtils function should do the trick.

Reply:
I have `tested ss-commit-linter.sh` by adding a new commit as we discussed some times it is necessary to insert HTML which can not be done by `HtmlUtils.setHtml()` because it escapes the HTML so it is obvious to use .`html()`